contracts/BosonRouter.sol

```diff
@@ -105,8 +122,9 @@ contract BosonRouter is
 
     /**
      * @notice Pause the Cashier && the Voucher Kernel contracts in case of emergency.
-     * All functions related to creating new batch, requestVoucher or withdraw will be paused, hence cannot be executed.
-     * There is special function for withdrawing funds if contract is paused.
+     * All functions related to creating requestCreateOrder, requestVoucher, redeem, refund, complain, cancelOrFault, 
+     * cancelOrFaultVoucherSet, or withdraw will be paused and cannot be executed.
+     * There is a special function for withdrawing funds if contract is paused.
```

Comment:
This probably refers to `withdrawEthOnDisaster` from cashier? Someone might be looking for this function in this file, which is a bit confusing.

Reply:
Included in next commit.